### PR TITLE
Photofication

### PR DESCRIPTION
**Pull request by wilkdasilk**

### CODE REVIEW

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -24,4 +24,34 @@ $(document).on('turbolinks:load', function(){
      $('#flash').fadeOut(1000);
    }, 1000);
 
+   // called on file upload, client-side
+   // source: http://stevenyue.com/blogs/validate-attachment-file-size-and-type-in-rails/
+   $(document).on('turbolinks:load', function(){
```
Comment (medinal) [inline app/assets/javascripts/application.js]: isn't this already happening inside a document.on('turbolinks:load' function?

File: app/views/posts/_post.html.erb [hunk 2]
```diff
@@ -1,8 +1,18 @@
-<div>
+<<<<<<< HEAD
```
Comment (medinal) [inline app/views/posts/_post.html.erb]: What's this?

Comment (wilkdasilk) [inline app/views/posts/_post.html.erb]: should be gone now, saw that too

File: app/assets/javascripts/application.js (hunk 1 above)
Comment (wilkdasilk) [inline app/assets/javascripts/application.js]: You are correct, sir! Fixed.

**Pull request merged**